sdk/storage/Azure.Storage.DataMovement.Files.Shares/src/DataMovementSharesExtensions.cs

```diff
@@ -125,5 +245,150 @@ internal static partial class DataMovementSharesExtensions
                     lastModifiedTime: info.Details.LastModified,
                     properties: properties));
         }
+
+        /// <summary>
+        /// Helper method to append attribute name to stringbuilder.
+        /// </summary>
+        /// <param name="attributeName">name of attribute</param>
+        /// <param name="stringBuilder">stringbuilder reference</param>
+        private static void AppendAttribute(string attributeName, StringBuilder stringBuilder)
+        {
+            stringBuilder.Append(attributeName);
+            stringBuilder.Append('|');
+        }
+
+        /// <summary>
+        /// Helper function for NtfsFileAttributes to convert to a string.
+        /// </summary>
+        /// <returns>string</returns>
+        public static string ToAttributesString(this NtfsFileAttributes attributes)
```

Comment:
Is there no common helper for this? Or helper in the base Files package? Also the reverse of this method.

Reply:
There is, but it's internal to just the base Shares package. So I would have to move it outside fo the `NtfsFileAttributes` class, move it into some kind of Shared Extension and then include it into this package.

Reply:
Hmm, may be worth it? Not really sure when we decide to do this versus not. It is a decent amount of code to copy/paste

Reply:
Done, moved to ShareModelsExtension.cs.